README.md

```diff
@@ -1 +1,135 @@
-# Portfolio-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./assets/images/coder.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Portfolio Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is created by HTML and CSS to clone Portfolio project with all details through Figma.
+
+
+## 🛠 Built With <a name="built-with"></a>
+Html and CSS.
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[CSS file is externally linked to HTML]**
+- **[Portfolio is designed using HTML CSS]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Code editor(VS), Web browser and Git.
+
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/kakalygaday/Portfolio-Project
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command: 
+Open the file with your default brower or use live server extension
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdy**
+
+- GitHub: [@githubhandle](https://github.com/kakalygaday)
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add more HTML content]**
+
+- [ ] **[Add more CSS content]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project put a star and follow me for more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my heartfelt gratitude to my code reviewer and standup team for their invaluable support, guidance, and encouragement throughout this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] I would suggest you to work on you I noticed that the link is currently broken. To maintain a professional readme, please double-check and ensure that the link is functioning properly. Keep up the great work!